cmd/chart/upgrade.go

```diff
@@ -144,3 +147,28 @@ func splitImageName(reposName string) (string, string) {
 	nameParts := strings.SplitN(reposName, ":", 2)
 	return nameParts[0], nameParts[1]
 }
+
+func getLatest(vv []*semver.Version, current *semver.Version) string {
+	sort.Sort(sort.Reverse(semver.Collection(vv)))
+	hasRootless := strings.Contains(current.Prerelease(), "rootless")
+
+	if len(vv) == 0 {
+		return current.String()
+	}
+
+	latest := vv[0]
+	if latest.Prerelease() != "" {
```

Comment:
Could you extract to a small function with a self-describing name and a comment, or add a comment?

Reply:
I extracted the logic to for retrieving the latest stable version into its own function i.e  `getLatestStableVersion`. I also added a comment to the block of code that loops through the list of versions to return find the latest stable version in case an unstable version is the selected as the latest. I felt I had addressed your comments with these changes. Please let me know if there was anything I misunderstood so I can send your recommended changes in an updated PR.